site/zenodo_rdm/curation/tasks.py

```diff
@@ -10,12 +10,39 @@
 
 from celery import shared_task
 from flask import current_app
+from flask_mail import Message
 from invenio_access.permissions import system_identity
 from invenio_rdm_records.proxies import current_rdm_records_service as records_service
 from invenio_search.engine import dsl
 
 from zenodo_rdm.curation.curators import EURecordCurator
 
+RESULT_EMAIL_BODY = """
+EU Record Curation Processed
+
+Finished at: {finished_at}
+Processed since: {since}
+Total Records Processed: {processed}
+Records Failed: {failed}
+Records Approved: {approved}
+Record IDs Approved:
+{records_moved}
+"""
+
+
+def _send_result_email(content):
+    """Send curation result as email."""
+    subject = f"EU Record Curation Processesed {datetime.now().date()}"
+    body = RESULT_EMAIL_BODY.format(finished_at=datetime.now(timezone.utc), **content)
```

Comment:
```suggestion
    body = RESULT_EMAIL_BODY.format(finished_at=datetime.now(timezone.utc).isoformat(), **content)
```

nit: not sure about the rest of the context if it's `str`-able

Reply:
I tested all of it on my local, seems to be working fine for everything.